docs/scripting/api-environment.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Engine Environment
+category: Documentation
+tags:
+    - stable
+---
+
+
+## Common problems
+
+- **SetTimeout support**
+
+What people often run into when starting out with Minecraft Scripting is the problem with timing, the standards for timing code as you may have noticed are the `setTimeout` and `setInterval` functions and their cancel functions.
+
+These standardized methods are used by the frame system where you can set the delay to millisecond precision, but minecraft uses a tick to process changes in the world, right? That's why even these methods are not and will not be available, instead we got the [`rutTimeout`](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/system#runtimeout) and [`rutInterval`](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/system#runinterval) system methods from version 1.19.70, which delay to the precision of one tick, you can read more about on [Microsoft Docs](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/system) or [Wiki Tutorial](./script-server.md#scheduling)
```

Comment:
Typo

`rutTimeout` -> `runTimeout`

[Wiki Tutorial](./script-server.md#scheduling), I had an error on my end, the beginning `.` shouldn't be there for links